apps/site/src/_pages/docs/1_developing-blocks.mdx

```diff
@@ -157,14 +157,8 @@ The four components of a `Subgraph` are:
 - `depths`: which `edges` were followed from the `roots` when resolving the query, and how far
 - `edges`: connections between things in the graph. For example, a 'link entity' connects it and two other entities via `hasRightEntity`
   and `hasLeftEntity` edges (conceptually, the link entity is in the middle with an entity on its left and on its right)
-- `vertices`: each thing (`vertex`, or `node`) which was encountered when starting from the roots and following the specified `edges
```

Comment:
I've opted to remove `node` here because I think introducing yet another term is confusing (even if "node" is fairly common in describing graphs), replaced it with "element"